src/Sql/Sql/Database Backup/Cmdlet/SetAzureSqlDatabaseLongTermRetentionBackup.cs

```diff
@@ -0,0 +1,216 @@
+        public virtual string Location { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the server.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ParameterSetName = UpdateBackupDefaultSet,
+            Position = 1,
+            HelpMessage = "The name of the Azure SQL Server the backup is under.")]
+        [ResourceNameCompleter("Microsoft.Sql/servers", "ResourceGroupName")]
+        [ValidateNotNullOrEmpty]
+        public string ServerName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the database.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ParameterSetName = UpdateBackupDefaultSet,
+            Position = 2,
+            HelpMessage = "The name of the Azure SQL Database the backup is from.")]
+        [ResourceNameCompleter("Microsoft.Sql/servers/databases", "ResourceGroupName", "ServerName")]
+        [ValidateNotNullOrEmpty]
+        public string DatabaseName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the resource ID of the LTR Backup to update.
+        /// </summary>
+        [Parameter(ParameterSetName = UpdateBackupByResourceIdSet,
+            Mandatory = true,
+            Position = 0,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The Resource ID of the Database Long Term Retention Backup to remove.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the backup name.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ParameterSetName = UpdateBackupDefaultSet,
+            ValueFromPipelineByPropertyName = true,
+            Position = 3,
+            HelpMessage = "The name of the backup.")]
+        [ValidateNotNullOrEmpty]
+        public string BackupName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the resource group to use.
+        /// </summary>
+        [Parameter(Mandatory = false,
+            ParameterSetName = UpdateBackupDefaultSet,
+            HelpMessage = "The name of the resource group.")]
+        [ResourceGroupCompleter]
+        public override string ResourceGroupName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the backup storage redundancy to assign to the Azure SQL Database
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The name of the service objective to assign to the Azure SQL Database.",
+            ParameterSetName = UpdateBackupDefaultSet)]
+        [ValidateNotNullOrEmpty]
+        public string BackupStorageRedundancy { get; set; }
+
+        /// <summary>
+        /// Defines whether it is ok to skip the requesting of rule removal confirmation
+        /// </summary>
+        [Parameter(HelpMessage = "Skip confirmation message for performing the action")]
+        public SwitchParameter Force { get; set; }
```

Comment:
SupportsShouldProcess=true is enable to skip confirmation message to set. See detailed information here: https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/design-guidelines/should-process-confirm-impact.md#shouldprocess
Please help to verify whether Force is necessary.